news/5561.hooks.rst

```diff
@@ -0,0 +1 @@
+changed PY_DYLIB_PATTERNS to contain *.so
```

Comment:
The changelog is meant for end users - not PyInstaller devs. This'll be meaningless to anyone who doesn't know what the `PY_DYLIB_PATTERNS` variable does (especially since the name `PY_DYLIB_PATTERNS` is misleading in that implies that these dylibs are Python related). You should instead be explaining the implications of this change. i.e `collect_data_files()` now no longer requires `.so` files to have a `lib` prefix.

Reply:
Also capital letters and full stops please, you savage!

Reply:
Haha ok I will implement this. Cheers